### PR TITLE
Greedy Algorithm to find Minimum number of Coins - using C

### PR DESCRIPTION
**Pull request by sushant-varanasi**
## Have you read the Contribution Guidelines?

yes

## Description

check #171

## Checklist
- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the README.md and link to my code.
 
 
## Related Issues (Mention Issue no. like #xxx)

Fixes Issue #171
<br/>
(Write your answer above.)

### CODE REVIEW

File: C/README.md [hunk 1]
```diff
@@ -66,6 +66,8 @@ Format:
 
 -[Merge Sort](Mergesort.c)
 
+-[Minimum number of denominations using Greedy algo approach](MinDenominations_GreedyAlgo.c)
```
Comment (sunilgknair051) [inline C/README.md:69]: I would add Indian currency here to be specific.
Thanks.

**Review by sunilgknair051 — CHANGES_REQUESTED** [review]

**Comment by sunilgknair051** [comment]
But, @sushant-varanasi , overall well commented code. Thank you for the effort.

File: C/README.md (hunk 1 above)
Comment (sushant-varanasi) [reply C/README.md:69]: Sure, I'm updating the readme.md file

**Comment by sushant-varanasi** [comment]
Requested change done

**Pull request merged**